index.js

```diff
@@ -5,19 +5,22 @@ const BugsnagTransport = require('winston-bugsnag').BugsnagTransport;
 const env = process.env.NODE_ENV || 'development';
 const VError = require('verror').VError;
 
-const podName = process.env.POD_NAME || 'myService-1';
-const values = podName.split('-');
+const podName = process.env.POD_NAME || 'default-service-1234567890-abcde';
+const deploymentTag = process.env.DEPLOYMENT_TAG || 'abcde'
+const serviceName = podName.substring(0, podName.length - 17); // Pod name : servicename +  '-' + 10 bytes numeric + '-' + 5 bytes alphanumeric string
 const serviceContext = {
```

Comment:
How is this `serviceContext` passed to the logger?

Reply:
@andela-anandaa Check out line 28 in the file.

Reply:
@JamesNyamu - good stuff! Thanks. Solid work 👍